GalaxyZooTouchTable/GalaxyZooTouchTable/Services/PanoptesRepository.cs

```diff
@@ -0,0 +1,33 @@
+﻿using PanoptesNetClient;
+using PanoptesNetClient.Models;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Threading.Tasks;
+
+namespace GalaxyZooTouchTable.Services
+{
+    public class PanoptesRepository : IPanoptesRepository
```

Comment:
`Repository` seems like a weird word to use here. Why not call this `PanoptesService` to be more consistent with your directory naming structure

Reply:
https://stackoverflow.com/questions/5049363/difference-between-repository-and-service-layer

Did some research. Looks like the `Repository` naming convention is for basic CRUD operations. According to this post, I think this class is more business logicky and would make more sense being named `Service`